core/jobs/src/main/java/com/cognifide/aet/job/common/datafilters/jserrorsfilter/JsErrorsFilter.java

```diff
@@ -74,34 +72,21 @@ public boolean apply(@Nullable JsErrorLog input) {
   }
 
   private boolean shouldFilterOut(JsErrorLog jse) {
-    return matchSourceFile(sourceFile, jse.getSourceName())
-            && matchStrings(errorMessage, jse.getErrorMessage())
-            && matchNumbers(line, jse.getLineNumber());
+    boolean sourceNotSpecified = sourceFile == null;
+    boolean sourceEquals = jse.getSourceName().equals(sourceFile);
+    boolean sourceEndsWith = jse.getSourceName().endsWith("/" + sourceFile);
+
+    return (sourceNotSpecified || sourceEquals || sourceEndsWith)
```

Comment:
It is possible by specifying a long path. For example this filter:
```xml
<js-errors-filter source="comparators/jserrors/jsFileWithError.js" />
```
is working and will not hide errors from `some/other/path/jsFileWithError.js`
Is it OK or do we need some changes here?